jwcrypto/jwk.py

```diff
@@ -17,6 +17,24 @@
 from jwcrypto.common import base64url_decode, base64url_encode
 from jwcrypto.common import json_decode, json_encode
 
+__all__ = [
+    'JWKTypesRegistry',
+    'ParmType',
+    'JWKParameter',
+    'JWKValuesRegistry',
+    'JWKParamsRegistry',
+    'JWKEllipticCurveRegistry',
+    'JWKUseRegistry',
+    'JWKOperationsRegistry',
+    'JWKpycaCurveMap',
+    'InvalidJWKType',
```

Comment:
all of the above are not public intrerfaces

Reply:
ok, I agree that `JWKUseRegistry`, `JWKOperationsRegistry`, `JWKpycaCurveMap` is not public, but `InvalidJWKType` seems to me to refer to public methods, as it allows to handle exceptions

Reply:
ok for InvalidJWKType